### PR TITLE
Feat 2090 : allow administration of org and members in admin panel

### PR DESCRIPTION
**Pull request by phpgeekfr**
allow administration of org and members in admin panel

### CODE REVIEW

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts [hunk 1]
```diff
@@ -92,15 +94,38 @@ export class OrganizationComponent implements OnInit {
       }
     }));
 
+    await this.loadMembers();
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts:97]: I find it a weird process. I would do it like that instead : 
```typescript
this.members$ = this.getMembers(); // Same as loadMember without updating the state.
```
Like that you can use the async pipe to deal with changeDetection.

Comment (GrandSchtroumpf) [reply libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts:97]: I'm glad to discuss this cding design if you disagree.

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts [hunk 2]
```diff
@@ -92,15 +94,38 @@ export class OrganizationComponent implements OnInit {
...
+
+    const queryFn1 = ref => ref.where('type', '==', 'joinOrganization').where('mode', '==', 'invitation').where('fromOrg.id', '==', this.orgId).where('status', '==', 'pending');
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts:99]: Could you do it in several lines ? It's quite difficult to read I think.

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts [hunk 3]
```diff
@@ -92,15 +94,38 @@ export class OrganizationComponent implements OnInit {
...
+  public declineInvitation(invitation: Invitation) {
+    this.invitationService.declineInvitation(invitation);
+  }
+
+  public deleteInvitation(invitation: Invitation) {
+    this.invitationService.remove(invitation.id);
+  }
+
+  private async loadMembers(){
     const members = await this.organizationService.getMembers(this.orgId);
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts:120]: Any reason not to use an observable ?

File: libs/admin/src/lib/admin-panel/pages/organizations/organizations.component.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <h1><a routerLink="/c/o/admin/panel/">Administration panel</a> > Organizations</h1>
 <section fxLayout="row" fxLayoutAlign="space-around start">
   <article fxLayout="column" fxLayoutAlign="center center">
-    <div>
+    <div fxLayout fxLayoutGap="16px">
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/organizations/organizations.component.html:4]: 👍

File: libs/admin/src/lib/admin-panel/pages/user/user.component.html [hunk 5]
```diff
@@ -8,30 +8,42 @@ <h1><a routerLink="/c/o/admin/panel/">Administration panel</a> > User > {{ userI
...
+          <h3>Other settings</h3>
+          <ul>
             <li>
-              <b class="bf-dark-text">Blockframes admin</b>
-              <mat-icon [svgIcon]="isUserBlockframesAdmin  ? 'accept': 'cross'"></mat-icon>
-              <button mat-flat-button color="primary" (click)="setBlockframesAdmin()">
-                <span>{{ isUserBlockframesAdmin? 'Remove from admins' : 'Set admin' }}</span>
-              </button>
+              <b class="bf-dark-text">Blockframes admin : </b>
+              <a class="blockframes-admin-link" (click)="setBlockframesAdmin()">
```
Comment (GrandSchtroumpf) [inline libs/admin/src/lib/admin-panel/pages/user/user.component.html:43]: You don't want to sur mat-button or somthing like that here ?

File: libs/organization/src/lib/organization/components/member-permissions/member-permissions.component.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/components/member-permissions/member-permissions.component.html:1]: Remove this line

File: libs/organization/src/lib/organization/pages/member/member.component.ts [hunk 7]
```diff
@@ -45,11 +46,10 @@ export class MemberComponent implements OnInit {
 
     this.isAdmin$ = this.permissionQuery.isAdmin$;
     this.isSuperAdmin$ = this.permissionQuery.isSuperAdmin$;
-    const id = this.query.getActiveId();
 
     if (this.permissionQuery.isUserAdmin()) {
-      const queryFn1 = ref => ref.where('type', '==', 'joinOrganization').where('mode', '==', 'invitation').where('fromOrg.id', '==', id).where('status', '==', 'pending');
-      const queryFn2 = ref => ref.where('type', '==', 'joinOrganization').where('mode', '==', 'request').where('toOrg.id', '==', id).where('status', '==', 'pending');
+      const queryFn1 = ref => ref.where('type', '==', 'joinOrganization').where('mode', '==', 'invitation').where('fromOrg.id', '==', this.orgId).where('status', '==', 'pending');
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/member/member.component.ts:51]: Same as above

File: libs/organization/src/lib/permissions/+state/permissions.service.ts [hunk 8]
```diff
@@ -74,4 +51,53 @@ export class PermissionsService extends CollectionService<PermissionsState> {
...
+      return error.message;
+    }
+  }
+
+  /** Update role of a member of the organization */
+  private async _updateMemberRole(uid: string, role: UserRole) {
+    const orgId = this.query.getActiveId();
+    const permissions = await this.getValue(orgId);
+
+    switch (role) {
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/permissions/+state/permissions.service.ts:86]: Why not doing `permissions.roles[uid] = role` ?

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts (hunk 3 above)
Comment (phpgeekfr) [reply libs/admin/src/lib/admin-panel/pages/organization/organization.component.ts:120]: Yes, I don't know how to perform this (2 calls) directly to db (I cannot use query)

**Pull request merged**